### PR TITLE
Salt shakers now instantly gib vulpkanins

### PR DESCRIPTION
**Pull request by monster860**
Hey, we already have one OP way to kill people in one click, why not add more?

:cl: monster860
add: Salt shakers now instantly gib vulpkanins
/:cl:

### CODE REVIEW

**Comment by Necaladun** [comment]
This can be closed.

@monster860 Knock it off. You won't be warned again.

**Comment by SamHPurp** [comment]
speedmerge

File: code/modules/food_and_drinks/food/condiment.dm [hunk 1]
```diff
@@ -130,6 +130,10 @@
 	list_reagents = list("sodiumchloride" = 20)
 	possible_states = list()
 
+/obj/item/weapon/reagent_containers/food/condiment/saltshaker/attack(mob/living/carbon/human/target, mob/living/user)
+	if(istype(target) && target.species.name == "Vulpkanin")
```
Comment (Crazylemon64) [inline code/modules/food_and_drinks/food/condiment.dm]: Please remove the check for species, salt is toxic regardless of species

**Comment by Draconic-saint** [comment]
chrome_2017-07-16_13-38-35

**Comment by taukausanake** [comment]
And here I thought it was the salt reagent doing the gibbing. This is just lazy now

**Comment by Spartan6** [comment]
@tigercat2000
Missing the high priority tag. Good catch.

**Comment by LittleBigKid2000** [comment]
IPCs:
Immune to pain
Don't breathe
Immune to toxins
No eye damage from welding
Can be healed with common tools

Vulpkanin:
Punches can make you bleed
Not totally blind in the dark if they're colorblind

Yeah definitely merge this

**Comment by taukausanake** [comment]
The not blind in the dark thing is only if they have colorblind, which I'm betting most don't. I mean it defaults to surgical eyes after all. You'd have a better argument if it defaulted to colorblind

**Comment by LittleBigKid2000** [comment]
@taukausanake You know I'm being sarcastic when I say "Yeah definitely merge this", right?

**Comment by taukausanake** [comment]
@LittleBigKid2000 I may not play or particularly like (or dislike) IPCs but I sure do process thoughts like a robutt would

**Comment by Vivalas** [comment]
Tfw server admins police the Github

**Pull request closed without merging**